mobile-save-for-later/src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala

```diff
@@ -55,6 +68,12 @@ object LambdaResponse extends Base64Utils {
   def apply(apiGatewayLambdaResponse: ApiGatewayLambdaResponse) : LambdaResponse = {
     LambdaResponse(apiGatewayLambdaResponse.statusCode, apiGatewayLambdaResponse.body, apiGatewayLambdaResponse.headers)
   }
+  def toHttp4sRes(lambdaResponse: LambdaResponse): Response[IO] = {
+    val body = lambdaResponse.maybeBody.map(b => Stream(b).through(utf8Encode)).getOrElse(EmptyBody)
+    val headersRes = Headers(lambdaResponse.headers.map({ case (k, v) => Header.Raw(CIString(k), v) }).toList)
+    Status.fromInt(lambdaResponse.statusCode).fold(_ => Response(InternalServerError, headers = headersRes, body = body),
+      status => Response(status, headers = headersRes, body = body))
+  }
```

Comment:
Love this method and `fromHttp4sRequest()` too - great helper functions! This one can be written a bit more concisely by without declaring the two `val`s, and not putting so much inside the status-fold.

```suggestion
  def toHttp4sRes(lambdaResponse: LambdaResponse): Response[IO] = Response(
    Status.fromInt(lambdaResponse.statusCode).getOrElse(InternalServerError),
    headers = Headers(lambdaResponse.headers.map({ case (k, v) => Header.Raw(CIString(k), v) }).toSeq),
    body = lambdaResponse.maybeBody.map(b => Stream(b).through(utf8Encode)).getOrElse(EmptyBody)
  )
```